src/commands/run.rs

```diff
@@ -16,20 +16,12 @@ use wasmtime_cli_flags::{CommonOptions, WasiModules};
 use wasmtime_wasi::maybe_exit_on_error;
 use wasmtime_wasi::sync::{ambient_authority, Dir, TcpListener, WasiCtxBuilder};
 
-#[cfg(any(
-    feature = "wasi-crypto",
-    feature = "wasi-nn",
-    feature = "wasi-threads",
-    feature = "wasi-http"
-))]
+#[cfg(any(feature = "wasi-nn", feature = "wasi-threads", feature = "wasi-http"))]
 use std::sync::Arc;
```

Comment:
Go ahead and merge this but I'm noticing that this `cfg` attribute should have been deleted entirely when my `GuestProfiler` work landed, since that requires `Arc` and is not behind any feature flags. I actually thought I had fixed that but the git history tells me I did not.